.vscode/launch.json

```diff
@@ -1,24 +1,29 @@
 {
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "name": "Debug Dev Binary",
-            "type": "go",
-            "request": "launch",
-            "mode": "exec",
-            "program": "${workspaceFolder}/.air/bytebase",
-            "args": [
-                "--port", "8080",
-                "--frontend-port", "3000",
-                "--data", "${workspaceFolder}/.air",
-            ]
-        },
-        {
-            "name": "Attach to Process",
-            "type": "go",
-            "request": "attach",
-            "mode": "local",
-            "processId": 0
-        }
-    ]
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Debug Dev Binary",
+      "type": "go",
+      "request": "launch",
+      "mode": "exec",
+      "program": "${workspaceFolder}/.air/bytebase",
+      "args": [
+        "--host",
```

Comment:
nit: I prefer the former style which keeps the parameter name and value at the same line. It's easier for eye parsing.

Reply:
Auto formatting makes this